codegen/table.go

```diff
@@ -1,11 +1,19 @@
 package codegen
 
 import (
+	"bytes"
+	"embed"
 	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"path"
 	"reflect"
+	"strings"
+	"text/template"
 
 	"github.com/cloudquery/plugin-sdk/schema"
-	"github.com/rs/zerolog"
+	"github.com/iancoleman/strcase"
```

Comment:
Is there anything preventing use from using the SDK's caser https://github.com/cloudquery/plugin-sdk/tree/b47c8eda53e7a4351b2901754d5b232d04fc7967/caser?